pkg/virt-controller/watch/vm_test.go

```diff
@@ -5492,6 +5492,68 @@ var _ = Describe("VirtualMachine", func() {
 					Expect(err).To(Succeed())
 					Expect(vm.Spec.Preference.RevisionName).To(Equal(expectedPreferenceRevision.Name))
 				})
+
+				It("should apply preferences to hot plugged interface in VMI", func() {
```

Comment:
For a follow up, consider moving all preferences tests to its own test file. The file size is too big.

Reply:
https://github.com/kubevirt/kubevirt/issues/10175 - Yeah I've been trying to sign up folks to this `good-first-issue` for ages now to help with this but everyone eventually gives up.

FWIW refactoring a large part of `pkg/instancetype` (the inherited interface, methods approach, single files etc)  is currently queued up in my backlog for the 1.4 cycle.